src/components/02_atoms/Widgets/EntityReference.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import keycode from 'keycode';
+import Downshift from 'downshift';
+
+import Paper from '@material-ui/core/Paper';
+import TextField from '@material-ui/core/TextField';
+import Chip from '@material-ui/core/Chip';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
+
+import api from './../../../utils/api/api';
+
+class EntityReference extends React.Component {
+  static propTypes = {
+    ...WidgetPropTypes,
+    inputProps: PropTypes.shape({
+      resourceIdentifer: PropTypes.string,
+    }).isRequired,
+  };
+
+  state = {
+    inputValue: '',
+    selectedItem: [],
+    suggestions: [],
+    loading: false,
+  };
+
+  handleChange = item => {
+    let { selectedItem } = this.state;
+
+    if (selectedItem.indexOf(item) === -1) {
+      selectedItem = [...selectedItem, item];
+    }
+
+    this.setState({
+      inputValue: '',
+      selectedItem,
+    });
+  };
+
+  handleInputChange = event => {
+    this.setState({ loading: true, inputValue: event.target.value }, () => {
+      api('taxonomy_term', {
+        queryString: {
```

Comment:
The current architecture abstracts the querying part outside of the widget. Should we open a follow-up to figure out if we have to implement that on the client side as well?